judo-runtime-core-jsl-itest/models/EnvironmentAndSequenceModel/src/test/resources/EnvironmentAndSequenceModel.jsl

```diff
@@ -85,3 +85,19 @@ entity SystemEnvironmentVariableWithExpression {
     field Timestamp timestampWithoutOffset default : Timestamp.getVariable(category = "ENVIRONMENT", key = "timestampWithoutOffset").plus(minutes = 720);
     field Time time default : Timestamp.of(date = `2022-11-22`,time = Time.getVariable(category = "ENVIRONMENT", key = "time")).plus(minutes = 60).time();
 }
+
+entity SequenceNames {
+    field Integer sequence1 <= Integer.getVariable(category = "SEQUENCE", key = "Xárvíztűrő 'tükörfúrógép");
+    field Integer sequence2 <= Integer.getVariable(category = "SEQUENCE", key = "ORDER");
+    field Integer sequence3 <= Integer.getVariable(category = "SEQUENCE", key = "SELECT");
+    field Integer sequence4 <= Integer.getVariable(category = "SEQUENCE", key = "GROUP BY");
+    field Integer sequence5 <= Integer.getVariable(category = "SEQUENCE", key = "@%££$");
+    field Integer sequence6 <= Integer.getVariable(category = "SEQUENCE", key = "+");
+    field Integer sequence7 <= Integer.getVariable(category = "SEQUENCE", key = "a");
+    field Integer sequence8 <= Integer.getVariable(category = "SEQUENCE", key = "+++");
+    field Integer sequence9 <= Integer.getVariable(category = "SEQUENCE", key = "3");
+    field Integer sequence10 <= Integer.getVariable(category = "SEQUENCE", key = " 333 ");
```

Comment:
Why were the test cases containing white spaces removed? (see original test cases from esm itest)